e2e/tests/cohorts/cohort-create.spec.ts

```diff
@@ -101,52 +101,45 @@ describe('Create Cohorts Test', () => {
     const dataPage = new WorkspaceDataPage(page);
     const cohortBuildPage = await dataPage.clickAddCohortsButton();
 
-    // Include Group 1: Demographics Age range: 21 - 90.
-    const minAge = 21;
-    const maxAge = 90;
+    // Include Group 1: Demographics Deceased.
     const group1 = cohortBuildPage.findIncludeParticipantsGroup('Group 1');
-    await group1.includeAge(minAge, maxAge);
-
+    await group1.includeDemographicsDeceased();
```

Comment:
Can we put the age item back here on group 1? That way we keep test coverage on the age slider and count won't be zero since it will be on the same group as Deceased.

Reply:
That's an excellent idea. Updated Group 1 to include both Age and Deceased criteria.